.travis.yml

```diff
@@ -33,10 +33,23 @@ before_script:
 
     - make extracheck
 
-script:
-    - make coverage
+os:
+  - linux
+  - osx
+  # - windows -- currently not built for windows
+
+matrix:
+  include:
+    - stage builds
+    script:
+      make build-natives
```

Comment:
here you could also skip the osx platform, as the https://github.com/mendersoftware/mender-artifact/blob/master/Dockerfile.binaries should produce this - e.g. using: `docker build -t mender-artifact-binaries -f Dockerfile.binaries .` and then just `docker run --rm --entrypoint "/bin/sh" -v $(pwd):/binary mender-artifact-binaries -c "cp /go/bin/mender-artifact-darwin /binary"`... so you would only use the osx platform to run the tests with the herein built binary

Reply:
I thought this was only supposed to check if it builds on osx, and not run the tests.

Reply:
well we could do both on travis - I am not sure what time it costs to spin up osx and if an instance can be reused between build and test, but the dockerfile could build mac (and potentially windows) builds to pass the result straight to a testing stage.

Reply:
I think that is contained in https://tracker.mender.io/secure/RapidBoard.jspa?rapidView=10&view=detail&selectedIssue=MEN-2505&quickFilter=67, so I will move to that next.

Reply:
yes, there we would definitely need an osx instance, probably won't hurt the build also runs on travis osx